internal/services/network/virtual_hub_connection_resource.go

```diff
@@ -151,6 +153,19 @@ func resourceVirtualHubConnectionSchema() map[string]*pluginsdk.Schema {
 				},
 			},
 		},
+
+		"vnet_local_route_override_enabled": { // keep it in root block to make default work
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			ForceNew: true,
+			Default:  false,
+		},
+
+		"static_route_propagate_enabled": { // keep it in root block to make default work
```

Comment:
the gramar here is wrong
```suggestion
		"static_route_propagation_enabled": { // keep it in root block to make default work
```